docs/integration/akka.md

```diff
@@ -5,6 +5,45 @@ description: Sematext APM monitoring integration for Akka actors, dispatchers, a
 
 - Instructions: [https://apps.sematext.com/ui/howto/Akka/overview](https://apps.sematext.com/ui/howto/Akka/overview)
 
+## Akka Default Alerts
+
+As soon as you create an Akka App, you will receive a set of default [alert rules](https://sematext.com/docs/guide/alerts-guide/). These pre-configured rules will [notify](https://sematext.com/docs/alerts/alert-notifications/) you of important events that may require your attention, as shown below.
+
+### CPU idle < 5%
+
+This alert rule continuously monitors the CPU idle percentage in an Akka system, triggering a warning (WARN priority) when the CPU idle falls below 5%. The minimum delay between consecutive alerts triggered by this alert rule is set to 10 minutes.
+
+Suppose an Akka system typically operates with CPU idle percentages above 5%. However, due to increased workload, the CPU idle percentage drops below 5%. When this happens, the alert rule  the alert rule triggers a warning.
+
+#### Actions to take
+
+- Review the resource usage of the Akka system to identify the cause of increased CPU usage and decreased idle percentage. This may involve analyzing application load, system configuration, and resource allocation
+
+### Network received anomaly
+
+This alert rule continuously monitors the rate of network data received by an Akka application, checking for anomalies in the received data rate over time. When anomalies are detected, it triggers a warning (WARN priority). The minimum delay between consecutive alerts triggered by this alert rule is set to 10 minutes.
+
+Suppose an Akka application typically receives network data at a consistent rate, but due to network congestion or other issues, the rate of data reception suddenly drops or spikes unexpectedly. When this happens, the alert rule triggers a warning.
+
+#### Actions to take
+
+- Check the status of the network infrastructure to determine if there are any issues such as congestion, packet loss, or connectivity problems affecting data reception
+- Review application logs and system metrics for any errors or warnings that may be impacting network data reception
+
+### Network transmitted anomaly
```

Comment:
This is fine, but maybe we can just say it's the same as ingress, but this is egress? :)

Reply:
Yep, I'm trying to give all the details for each alert rule. There is a bunch of repeated text, but it's like each piece of info stands alone and helps connect with SC later on.

Reply:
Got it. Thanks!